spack.yaml

```diff
@@ -4,7 +4,7 @@
 # configuration settings.
 spack:
   specs:
-    - access-esm1p5@git.2024.05.1
+    - access-esm1p5@git.2024.05.2
```

Comment:
It's only a *really* minor point but I'd suggest bumping the version to 2024.12.0 because the orbital parameter setting is a substantial change. I think it gives the user a little bit of information to suggest a big change.

Note it's totally just my opinion, we have no hard and fast rules.

Reply:
can do!